controllers/keystoneservice_controller.go

```diff
@@ -48,11 +48,6 @@ func (r *KeystoneServiceReconciler) GetKClient() kubernetes.Interface {
 	return r.Kclient
 }
 
-// GetLogger -
-func (r *KeystoneServiceReconciler) GetLogger() logr.Logger {
-	return r.Log
-}
-
```

Comment:
GetClient, GetKClient and GetScheme could also be removed. its a relikt from before we introduced the helper. Could also do it in a follow up.

Reply:
sure , as a follow up